doc/admin-guide/layer-4-routing.en.rst

```diff
@@ -0,0 +1,124 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+
+.. include:: ../common.defs
+.. default-domain:: c
+.. _layer-4-routing:
+
+Layer 4 Routing
+**************************
+
+|TS| supports a limited set of layer 4 routing options. In this case |TS| acts effectively as a
```

Comment:
In this case*,*

Reply:
Changed to `In such use`, which I think is clearer.